src/mods/20190509091500-issue-18-add.yml

```diff
@@ -0,0 +1,10 @@
+modifications:
+- action: ADD
+  parentCode: RYF-QPR
+  parentName: Sport
```

Comment:
Shouldn't this be under "Sporter" and not "Sport"?

Reply:
moved it!, good catch!